flow/peerdbenv/dynamicconf.go

```diff
@@ -76,6 +76,12 @@ END;`, ValueType: protos.DynconfValueType_STRING,
 		ApplyMode:        protos.DynconfApplyMode_APPLY_MODE_AFTER_RESUME,
 		TargetForSetting: protos.DynconfTarget_ALL,
 	},
+	{
+		Name: "PEERDB_NULLABLE", DefaultValue: "false", ValueType: protos.DynconfValueType_BOOL,
```

Comment:
I thought we want it be true by default? 
cc @iskakaushik @saisrirampur

Reply:
shouldn't be, perf impact
needs to be opt-in

Reply:
also prefer to have new options not change existing behavior